src/gdcdictionary/schemas/_terms.yaml

```diff
@@ -1959,9 +1959,10 @@ encoding:
       term: >-
         Encoding
       source: FastQC
-      cde_id:
+      cde_id: Encoding
```

Comment:
This looks a bit odd as a cde_id.  Is this something new or a misplaced term?  Same with "Total Sequences"

Reply:
We draw the definitions for Encoding and Total Sequences from this [site](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/3%20Analysis%20Modules/1%20Basic%20Statistics.html), so there is no formal cde_id or NCIt code. This was mainly to address that these properties have links that are identified as 'null' in the Viewer page for read_group_qc node: https://docs.gdc.cancer.gov/Data_Dictionary/viewer/#?view=table-definition-view&id=read_group_qc  

Side note, these properties belong to a node that is not immediately found on the overall View page (https://docs.gdc.cancer.gov/Data_Dictionary/viewer/), therefore more for internal use.

Reply:
@wwysoc2 came across that NCIt _did_ have entries for those two, updated their definitions and codes.